### PR TITLE
New rules for last seen in 45 and 60 days

### PR DESCRIPTION
**Pull request by eanders**

### CODE REVIEW

File: spec/models/rules/seen_in_last_forty_five_days_spec.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe Rules::SeenInLastFortyFiveDays, type: :model do
+  describe 'clients_that_fit' do
+    let!(:rule) { create :seen_in_last_forty_five_days }
+
+    let!(:bob) { create :client, first_name: 'Bob', calculated_last_homeless_night: 44.days.ago.to_date }
+    let!(:roy) { create :client, first_name: 'Roy', calculated_last_homeless_night: 46.days.ago.to_date }
```
Comment (awisema) [inline spec/models/rules/seen_in_last_forty_five_days_spec.rb:8]: Add a boundary condition test? (45 days for this spec).
